src/utils.tsx

```diff
@@ -288,16 +288,16 @@ export const sendPerfTiming = ({
   _tracking = tracking,
   actualDuration,
   id,
-  phase,
+  url,
 }: {
   _tracking?: typeof tracking;
   actualDuration: number;
   id: string;
-  phase: string;
+  url: string;
 }) => {
   _tracking.timing({
     category: 'renderPerf',
-    label: phase,
+    label: url,
```

Comment:
We could probably set the value of `label: window.location.href`.

Reply:
Make that the default, or just always set it to `window.location.href`? I guess the latter is okay for now, as I see no reason we'd need to specify a different url for any of the current use cases.

Reply:
Just thinking if that will make the test cases more complicated. I guess we can just test the value of `url` in the test for `sendPerfTiming` and not even bother checking it for the tests for `CodeView` and `DiffView`.

Reply:
We don't necessarily need to test the value of the `url`. I see this value as an implementation detail of the `sendPerfTiming` function.

Reply:
Interesting. So even though there will be logic that sets the value of the `url` based on `window.location`, you don't think we should have a test anywhere that verifies that logic? Or are you just agreeing that we only need to test it in the test for `sendPerfTimimg`?

Reply:
> Or are you just agreeing that we only need to test it in the test for `sendPerfTimimg`?

That, indeed. sorry if that wasn't clear, we should have only one test case.